spec/jobs/bcc_post_spec.rb

```diff
@@ -54,5 +56,15 @@
 
       expect(Topic.count).to eq(topic_count + 2)
     end
+    
+    it 'works with username and email personalization' do
+
+      ::Jobs::BccPost.new.execute(user_id: sender.id, create_params: create_params.merge("raw": "this is the content I want to send to {username}", target_emails: 'test@test.com'))
+
+      raw = Topic.last.excerpt
+
+      expect(raw).to include(user0.username).or include(user1.username).or include("test@test.com")
```

Comment:
This is a bit odd, why `or` here?  I feel that could fail easily.

Instead I would make sure a post with "I want to send to #{user0.username}", etc. exists

Reply:
Modified the method to check if a post with such text exists.